api/src/main/scala/pl/touk/nussknacker/engine/api/Context.scala

```diff
@@ -64,4 +64,7 @@ case class Context(id: String, variables: Map[String, Any], parentContext: Optio
   def popContext : Context =
     parentContext.getOrElse(throw new RuntimeException("No parent context available"))
 
+  def clearRedundantVariablesExcept(variablesToLeave: Set[String]) : Context =
```

Comment:
Can we have a method like `clearUserVariables`? or sth like that? And have EventTimestampVariableName in that method?

Reply:
changed it, please check if it's okay now

Reply:
Let's add a comment that it leaves technical variables, hidden from user